evap/static/ts/src/student-vote.ts

```
@@ -176,3 +180,242 @@ function scrollToFirstChoiceError() {
         fancyFocus(findCorrectInputInRow(tabRow));
     }
 }
+
+const dataElement = document.querySelector<HTMLElement>(".dataElement")!;
```

Please use `assertOrError` (so that we can fail early and facilitate debugging)